models/biocro/R/model2netcdf.BIOCRO.R

```diff
@@ -78,7 +79,7 @@ model2netcdf.BIOCRO <- function(result, genus = NULL, outdir, lat = -9999, lon =
                        Evap = udunits2::ud.convert(SoilEvaporation + CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
                        TVeg = udunits2::ud.convert(CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
                        LAI = LAI))
-    
+    RR$NPP <- udunits2::ud.convert(c(0, diff(R$TotLivBiom)), "kg/m2/h", "kg/m2/s")
```

Comment:
1. Got it, and making a note to check the standard any time I touch an output converter.

2. It looks like this is biomass not C -- note that the k multiplier includes a division by `c2biomass` ( == 0.4).